detox/local-cli/testCommand/builder.js

```diff
@@ -135,5 +135,16 @@ module.exports = {
     group: 'Debugging:',
     describe: '[Jest Only] Allows debugging of the underlying test runner',
     boolean: true,
+  },
+  'repl': {
```

Comment:
We have to consider here too the potential case of using `repl` alongside `inspect-brk` ==> Throw error / be verbose about what's going to happen (I prefer the former)